### ISSUE TITLE
Debian8 DNSMASQ and AMI

### ISSUE THREAD

**Issue by df-sh** [jlund/streisand#172]
I have been testing out several different AMI#s for Deb8 but with some minor changes to the dnsmasq restart problem, I have been able to spin up amazon servers in t2.micro for https://aws.amazon.com/marketplace/pp/B00WUNJIEE

After modifying the AMI list for amazon debian 8.1 from this product https://aws.amazon.com/marketplace/pp/B00WUNJIEE

![deb8](https://cloud.githubusercontent.com/assets/12028761/8370857/eb50e9cc-1c00-11e5-9932-71a8a8ec1f26.PNG)

in the file streisand\playbooks\roles\genesis-amazon\defaults\main.yml
```
---
aws_debian7_california_ami_id: "ami-a5d621e1"
aws_debian7_ireland_ami_id: "ami-0ed89d79"
aws_debian7_oregon_ami_id: "ami-3d56520d"
aws_debian7_sao_paulo_ami_id: "ami-a5bd3fb8"
aws_debian7_signapore_ami_id: "ami-60989c32"
aws_debian7_sydney_ami_id: "ami-07e3993d"
aws_debian7_tokyo_ami_id: "ami-fe54f3fe"
aws_debian7_virginia_ami_id: "ami-8b9a63e0"
aws_instance_type: "t2.micro"
```

I deleted the the dnsmasq restart notify on line 153 of the openvpn tasks
of file streisand\playbooks\roles\openvpn\tasks\main.yml

```
- name: Copy dnsmasq configuration file into place
  copy: src=etc_dnsmasq.conf
        dest=/etc/dnsmasq.conf
  notify: Restart dnsmasq
```

The script completed... havent tested the server much after that...

**Comment by jlund** [comment]
If you could, that would be really helpful. I'm hoping  that only minor changes will be necessary. Thank you for all of your contributions so far!

**Comment by df-sh** [comment]
Ok, so I tried spinning up about 6 different debian 8.1 builds using AMI's from t2 micro and t1 micro listed.

Script runs fine for amazon...  normal usage for shadowsocks and vpn seems to be working for me

the biggest change in setup is the new init system in deb8 so the restart for dnsmasq I posted is really the only change... I dont use openvpn so I am not sure if not restarting dnsmasq at that point in the playbook would make any changes.  

Someone more familiar with the new init system would be better suited to that task

**Comment by c0state** [comment]
I tried this on DigitalOcean using the Debian 8.1 x64 image.

I had to comment out

    name: Make sure that nothing is listening on port 80 on the public interface

in the file

    playbooks/roles/nginx/tasks/main.yml

as it would timeout for some reason.  This is in addition to the dnsmasq change mentioned by @df-sh 

L2TP/IPsec doesn't seem to work.  

Strongswan 5.x (used in Debian 8.1) changed significantly (at least architecturally) versus 4.x (which Debian 7 uses) as per https://www.strongswan.org/blog/2012/06/20/bye-bye-pluto.html
